x/permission/keeper/keeper.go

```diff
@@ -287,8 +287,9 @@ func (k Keeper) VerifyPolicy(ctx sdk.Context, resourceID math.Uint, resourceType
 				if memberFound {
 					if effect == types.EFFECT_ALLOW {
 						allowed = true
+						withGroupID = item.GroupId
```

Comment:
Can we break here?

Reply:
Nope. We can't. Because if still has a group that contains this operator has deny permission of the resource, it will deny.